src/styles/blocks/product-card.css

```diff
@@ -0,0 +1,102 @@
+.product-card {
+  height: 406px;
+  width: 198px;
```

Comment:
2 points:
- height — no, you don't need it, content spreads all over available space and dictate height to its parent
- width — should be 200px (hint: what is box-sizing for?)